client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog.jsx

```diff
@@ -89,9 +101,20 @@ module.exports = React.createClass( {
 				buttons={ deactivationButtons }
 				onClose={ this.close }
 				transitionLeave={ false }>
-				<h1>{ this.translate( 'Disconnect Jetpack' ) }</h1>
+				<h1>{ translate( 'Disconnect Jetpack' ) }</h1>
 				<p>{ moreInfo }</p>
 			</Dialog>
 		);
 	}
-} );
+}
+
+export default connect(
+	state => ( {
+		selectedSite: getSelectedSite( state )
+	} ),
+	null,
```

Comment:
Let's pull in [recordEvent](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics):
```
//in onClose
this.props.recordEvent( 'Jetpack', 'Clicked To Cancel Disconnect Jetpack Dialog' );
//...
export default connect(
    ( state ) => ( {
        selectedSite: getSelectedSite( state )
    } ),
    {
        recordEvent
    },
    null,
    {
        withRef: true
    }
)( localize( DisconnectJetpackDialog ) )
```